target_quickbooks/client.py

```diff
@@ -458,10 +465,16 @@ def request_api(self, http_method, endpoint=None, params={}, request_data=None,
         resp = self._request(http_method, endpoint, params, request_data, headers, verify=verify, stream=stream)
         return resp      
 
+    @backoff.on_exception(
+        backoff.expo,
+        (ClientRateLimitError, Server5xxError),
+        max_tries=10,
+        factor=3
```

Comment:
The [docs](https://developer.intuit.com/app/developer/qbo/docs/learn/rest-api-features#limits-and-throttles:~:text=You%E2%80%99ll%20see%20HTTP%20Status%20Code%20429%20when%20throttling%20occurs.%20If%20you%20see%20this%2C%20wait%2060%20seconds%20before%20retrying%20the%20request.) says to wait for 60 seconds in case we hit a 429. 
By using `backoff.expo` with `factor=3` the delays for each backoff would be `3, 6, 12, 24, 48, 96, 192, 384, 768, 1536` only the the sixth consecutive backoff would be greater than 60 seconds.

I suggest changing it to `factor=60, base=1.3`, that way the backoff times would be `60, 78, 101, 131, 171, 222, 289, 376, 489, 636`

Reply:
Interesting, I copied the same thing from tap-quickbooks.